packages/allure-codeceptjs/src/index.ts

```diff
@@ -219,6 +221,8 @@ class AllureReporter {
     this.currentTest = null;
     // @ts-ignore
     if (test.state === "failed" && test._retries !== 0 && currentRetry !== test._retries) {
+      // @ts-ignore
+      test.parent.ctx.currentTest = test;
```

Comment:
`Allure reporter` used purely to reporting info. But in this place you modify data. It can cause problems in the future then you using other plugins.

Reply:
@vovsemenv without this change in Report test data in retries missed, and we have only first attempt try

Reply:
**with change** take a look on **Description** part:
 main run:
<img width="521" alt="Screenshot 2023-08-03 at 14 03 12" src="https://github.com/allure-framework/allure-js/assets/4963181/7c3bed87-1574-4b5e-87ef-6cc104470dee">
attempt1
<img width="609" alt="Screenshot 2023-08-03 at 14 03 47" src="https://github.com/allure-framework/allure-js/assets/4963181/1cdc4c1e-9dd1-4c9d-8ce5-237087958e9e">
attempt2:
<img width="762" alt="Screenshot 2023-08-03 at 14 12 24" src="https://github.com/allure-framework/allure-js/assets/4963181/a078122e-0735-4675-a2c6-01ecff4bb51f">

**without change**:
main run:
<img width="1318" alt="Screenshot 2023-08-03 at 14 09 41" src="https://github.com/allure-framework/allure-js/assets/4963181/b26fa63c-153b-4de8-806f-16da9b70513f">

 attempt1
<img width="958" alt="Screenshot 2023-08-03 at 14 09 22" src="https://github.com/allure-framework/allure-js/assets/4963181/1bc865da-9b1a-4ed3-bb7c-384f6691b7c6">

attempt2
<img width="781" alt="Screenshot 2023-08-03 at 14 08 45" src="https://github.com/allure-framework/allure-js/assets/4963181/6f76b92d-b37d-4ea0-b8ea-d01d65c0283d">